spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/reactive/DefaultPartHttpMessageReaderCustomizerConfiguration.java

```diff
@@ -0,0 +1,65 @@
+package org.springframework.boot.autoconfigure.web.reactive;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.file.Paths;
+import java.util.function.Consumer;
+
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.boot.context.properties.PropertyMapper;
+import org.springframework.boot.web.codec.CodecCustomizer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.annotation.Order;
+import org.springframework.http.codec.multipart.DefaultPartHttpMessageReader;
+import org.springframework.util.unit.DataSize;
+
+@Configuration(proxyBeanMethods = false)
+@EnableConfigurationProperties(MultipartProperties.class)
+class DefaultPartHttpMessageReaderCustomizerConfiguration {
```

Comment:
Naming... 

So, part of me (no pun intended - terrible, terrible) wants to name this more generically like `MultipartConfiguration` but on the other hand it is currently very specific to what the target of configuration is - `DefaultPartHttpMessageReader`.

Reply:
Ha! The only other option I'm seeing is `ReactiveMultipartConfiguration` which doesn't leak implementation details and is specific enough - it's also in line with the `Properties` class. Once we're done with the overall implementation we can always look at the Servlet equivalent and check for inconsistencies.